packages/xrpl/src/models/ledger/PermissionedDomain.ts

```diff
@@ -0,0 +1,29 @@
+import { AuthorizeCredential } from '../common'
+
+import { BaseLedgerEntry, HasPreviousTxnID } from './BaseLedgerEntry'
+
+export default interface PermissionedDomain
+  extends BaseLedgerEntry,
+    HasPreviousTxnID {
+  /* The ledger object's type (PermissionedDomain). */
+  LedgerEntryType: 'PermissionedDomain'
+
+  /* The account that controls the settings of the domain. */
+  Owner: string
+
+  /* The credentials that are accepted by the domain.
+  Ownership of one of these credentials automatically
+  makes you a member of the domain. */
+  AcceptedCredentials: AuthorizeCredential[]
```

Comment:
This type name is really confusing here - can we alias it or something?

Reply:
I had to use the `AuthorizeCredential` name because I imported it from the Credential-related file. Please check the earlier PR review comments from @achowdhry-ripple in this page

Reply:
Do you have a preference on the alias name?